.gitignore

```diff
@@ -34,6 +34,9 @@ profile.cov
 # IdeaIDE
 .idea
 
+# VS Code
+.vscode
```

Comment:
@karalabe @fjl is it ok to extend .gitignore with the vscode hidden directory? Outside this,  this is a swarm-only PR which I would approve.

Reply:
:+1: